pkg/config/ebpf_tracer.go

```diff
@@ -59,4 +59,7 @@ type EBPFTracer struct {
 
 	// Enables debug printing of the protocol data
 	ProtocolDebug bool `yaml:"protocol_debug_print" env:"BEYLA_PROTOCOL_DEBUG_PRINT"`
+
+	// Enables Java instrumentation with the OpenTelemetry JDK Agent
+	UseOTelSDKForJava bool `yaml:"use_otel_sdk_for_java" env:"BEYLA_USE_OTEL_SDK_FOR_JAVA"`
```

Comment:
Reading the code above, I assume that enabling this would only generate OTEL metrics or traces. I think we should validate that if Prometheus is enabled this option can't be enabled.

Reply:
I think this happens right now:
https://github.com/grafana/beyla/pull/1704/files#diff-d6000537195fb5cd7aa9e0b6bea705c332ce53d5db9e070e39dffcdc191562e3R65

With prometheus scrape Metrics will be false, so we'll program it to send traces only.